features/window.yml

```
@@ -0,0 +1,30 @@
+name: Window
+description: "The Window object is home to a variety of functions, namespaces, objects, and constructors. Some of these are associated with the user interface window the document is loaded in, but many are items that need to be available in the global scope."
```

To create some symmetry with the `navigator` feature you created previously:

```suggestion
description: "The `window` global object represents a browser tab or iframe and its relationship to the document and other tabs. It's also a generic global object, under which many other APIs are located."
```